lib/views.js

```diff
@@ -269,12 +269,13 @@ exports.places_by_type_parent_id_name = {
 exports.contacts_by_depth = {
   map: function(doc) {
     if (['person', 'clinic', 'health_center', 'district_hospital'].indexOf(doc.type) !== -1) {
+      var value = doc.patient_id || doc.place_id;
```

Comment:
This looks to work if `doc` has _either_ `.patient_id` _or_ `.place_id`, but not both.  Is that acceptable?